src/components/Map/CircleComponent.tsx

```diff
@@ -1,23 +1,45 @@
+/* eslint-disable multiline-ternary */
```

Comment:
```suggestion
```
This is not needed. You don't use any ternary expression in this file so disabling this rule makes no changes.